### PR TITLE
[DROOLS-1426] remove CDI extension from drools-compiler

### PR DESCRIPTION
**Pull request by mariofusco**

### CODE REVIEW

File: drools-examples-api/ruleunit/target/surefire-reports/TEST-org.drools.example.api.reactivekiesession.RuleUnitExampleTest.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mswiderski) [inline drools-examples-api/ruleunit/target/surefire-reports/TEST-org.drools.example.api.reactivekiesession.RuleUnitExampleTest.xml]: is this here by accident?

Comment (mswiderski) [reply drools-examples-api/ruleunit/target/surefire-reports/TEST-org.drools.example.api.reactivekiesession.RuleUnitExampleTest.xml]: and few more ...

**Review by mswiderski — COMMENTED** [review]
there is a need to add drools-cdi dependency to https://github.com/droolsjbpm/jbpm/tree/master/jbpm-services/jbpm-services-cdi as it uses CDIHelper class that was moved + reorganisation of imports

**Review by mswiderski — APPROVED** [review]
looks good to me

File: drools-cdi/pom.xml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  </dependencies>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>findbugs-maven-plugin</artifactId>
+          <configuration>
+            <excludeFilterFile>${project.basedir}/src/main/findbugs/findbugs-exclude.xml</excludeFilterFile>
```
Comment (psiroky) [inline drools-cdi/pom.xml:126]: This needs to be removed as the file does not exist.

**Pull request merged**